README.md

```diff
@@ -312,10 +313,30 @@ tramway_navbar title: 'Purple Magic' do |nav|
 end
 ```
 
-### Tailwind components
+### Tailwind-styled forms
 
 Tramway uses [Tailwind](https://tailwindcss.com/) by default. All UI helpers are implemented with [ViewComponent](https://github.com/viewcomponent/view_component).
 
+#### tramway_form_for
+
+Tramway provides `tramway_form_for` helper that renders Tailwind-styled forms by default.
+
+```ruby
+= tramway_form_for User.new do |f|
+  = f.text_field :text
+  = f.password_field :password
+  = f.file_field :file
+  = tailwind_submit_button "Create User"
```

Comment:
Do you really need to leak rendering specifics(that this button is rendered with classes from tailwind) to the logic level?

Maybe better is `tramway_submit_button` or even `f.submit_button`

Reply:
You're right. Moved it to `f.submit`